index.html

```diff
@@ -2097,9 +2095,6 @@ <h3>Retrieve Verification Method</h3>
 Let <var>vmIdentifier</var> be set to <var>verificationMethod</var>.
           </li>
           <li>
-Let <var>vmPurpose</var> be set to <var>proofPurpose</var>.
-          </li>
-          <li>
 If <var>vmIdentifier</var> is not a valid URL, an error MUST be raised and SHOULD
```

Comment:
Thanks for taking a stab at this, @dlongley.  That said, this seems to add more than it removes, kind of defeating the purpose of #12, which was intended to remove unnecessary text.

In particular, it adds `vmRelationship` and `verificationRelationship`, which previously didn't exist in any of the specs.

Could you try again, just tweaking the site where you made comments starting with "This shouldn't be removed" applying one of your original suggestions, and retracting the new additions?  Thanks.

Reply:
> In particular, it adds vmRelationship and verificationRelationship, which previously didn't exist in any of the specs.

I believe this variable was previously specific to the DI specs (using `proofPurpose`) and my changes generalize it to better work with the existing section of the controller document:

https://w3c.github.io/controller-document/#verification-relationships

It is most definitely used by DI and by DID core: https://www.w3.org/TR/did-core/#verification-relationships

So the fix here is to ensure that a VM will only be retrieved if it matches the expected verification relationship. The DI spec just provides this value via the `proofPurpose` property of a proof. It could come from another place (or just be omitted and use the default relationship of `verificationMethod`) for callers of the algorithm that do not have a requirement for a more specific relationship.

Reply:
A tweak to @dlongley's suggestion
```suggestion
Let <var>vmRelationship</var> be set to <var>verificationRelationship</var>, if
present; otherwise, let it be set to the string `verificationMethod`.
          </li>
          <li>
If <var>vmIdentifier</var> is not a valid URL, an error MUST be raised and SHOULD
```